### ISSUE TITLE
Atom CLI triggers OS X Sierra native tabs

### ISSUE THREAD

**Issue by Moeriki** [atom/atom#12690]
### Prerequisites
- [X] Can you reproduce the problem in [safe mode](http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode)?
- [X] Are you running the [latest version of Atom](http://flight-manual.atom.io/hacking-atom/sections/debugging/#update-to-the-latest-version)?
- [X] Did you check the [debugging guide](http://flight-manual.atom.io/hacking-atom/sections/debugging/)?
- [X] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
- [X] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
- [X] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if your bug or enhancement is already reported?

For more information on how to write a good [bug report](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-bug-report) or [enhancement request](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-enhancement-suggestion), see the `CONTRIBUTING` guide.
### Description

When opening a second atom window from the CLI it triggers OS X Sierra native tabs in some case(s?). I could reproduce this in safe mode and after a factory-defaults reset.

This may or may not be an Electron issue though I could not produce it in another Electron app.
### Steps to Reproduce
1. have a 2 monitor setup
2. open terminal (iTerm in my case) full screen on monitor 2
3. in terminal `cd` to any project
4. in terminal `atom .`
5. atom opens on monitor 1 (**instance 1**)
6. put **instance 1** in OS X fullscreen mode if it is not and keep it in foreground on monitor 1
7. in terminal `cd` to any other project
8. in terminal `atom .`

**Expected behavior:** Atom opens **instance 2** in a second window (fullscreen optional)

**Actual behavior:** **instance 2** opens in a native tab

![atom-native-tabs](https://cloud.githubusercontent.com/assets/38758/18503304/7d5b6e40-7a5c-11e6-9ae4-f7ae56d8eb23.png)

**Work-around:** in step 6, make sure atom is nowhere in foreground

**Notes:** AFAIK issue could not be produced with one monitor, considering you need atom in fullscreen foreground to trigger this issue. If you have a terminal that can overlay your fullscreen Atom I'm guessing you can also produce this issue.
### Versions

macOS Sierra 10.12 GM

```
Atom    : 1.10.2
Electron: 0.37.8
Chrome  : 49.0.2623.75
Node    : 5.10.0
```

**Comment by Moeriki** [comment]
Fixed it.

System Preferences > Dock > Prefer tabs when opening documents > `manually` (default is `In fullscreen mode`)

![dock](https://cloud.githubusercontent.com/assets/38758/18503658/7750a6c6-7a5e-11e6-8056-d12875d0bf33.png)

**Comment by thomasjo** [comment]
@zcbenz Can we prevent this functionality entirely, regardless of the user's system preferences setting?

**Comment by danielbayley** [comment]
Maybe there should be an **option** to replace Atom tabs with the native tabs? There are already a few syntax themes that try to emulate them anyway…

**Comment by zcbenz** [comment]
@thomasjo There should be a way to disable it completely, but I will need to read Apple's documentation first.

Refs #6124.

**Comment by Ben3eeE** [comment]
@zcbenz Heads up, correct ref is https://github.com/electron/electron/issues/6124 I believe.

**Comment by thomasjo** [comment]
@Ben3eeE Hehe, I literally just edited his comment with the correct link 😄

**Comment by davibe** [comment]
I love this behaviour and i consider it a feature. I love to be able to finally open multiple projects in the same window and have sub-tabs for each of them. I don't need to struggle with multiple windows if i don't want to (i understand the multiple screen thing). I think Atom should support native tabs by adding menu items and shortcuts to cycle through them (I can only use the mouse at the moment)

**Comment by kevinsawicki** [comment]
This is being addressed in https://github.com/electron/electron/pull/7415

**Comment by miketamis** [comment]
This "randomly" happened to me, and it was really awesome, but then I couldn't figure out how to get into it again. I google how to get into it only to find this ticket. While I agree its a bug an should be fix, I think should be an easy way to get into it if it is what you want. (makes alt-tabbing to Atom so much nicer, because you keep all your atoms under the same window)

**Comment by intemperie** [comment]
+1 for keeping it as a feature. As @davibe said, it's very useful to support this really nice system feature. Let the user decide whatever they want :)

**Comment by OmgImAlexis** [comment]
@Moeriki as per #13526 this can be reproduced in single monitor mode as well.

**Comment by danielbayley** [comment]
I noticed that https://github.com/electron/electron/pull/9052 (Native tabs on macOS) was recently merged… so how will this fit into Atom?

**Comment by 50Wliu** [comment]
#12696 includes that PR.

**Comment by michaelaye** [comment]
Note that on macOS 10.12.4 this has nothing to do with fullscreen or not, it's simply always happening when `atom` wants to create a new window. And it's NOT a feature at all, because it pretty much screws up the display of 2 projects up and makes it very much unreadable:

<img width="861" alt="screenshot 2017-04-07 12 57 43" src="https://cloud.githubusercontent.com/assets/69774/24815262/cfe9d3b4-1b91-11e7-8945-1f2b6ecf2196.png">

**Comment by michaelaye** [comment]
But I'm guessing if it occurs always or only in fullscreen simply depends on the Dock tabs setting ("Always", or "Fullscreen only").

**Comment by mattdarveniza** [comment]
Yeah I actually really dig this feature for handling multiple projects, have noticed that the window menu is missing the "merge all windows" option that most other apps with native tabs has though. Adding in both the option to disable it entirely as well as actually supporting it properly would be great.

**Comment by mvaneijgen** [comment]
I loved this featured and now it is gone. Is their a way to get it back? MacOS 10.12.6, Atom 1.19.5 Prefer tabs when opening documents: Always.

**Comment by corbt** [comment]
Another vote for reenabling this. Makes my window organization much more sane.

**Comment by frankhock** [comment]
+1 for adding tabs back.

**Comment by rsese** [comment]
@mvaneijgen @corbt @frankhock - please see https://github.com/atom/atom/issues/15202, native tabs will be [available again in 1.21](https://github.com/atom/atom/issues/15202#issuecomment-327204426).
